modules/create-resource-init-bundle.adoc

```diff
@@ -56,14 +56,16 @@ If you are installing by using Helm charts, do not perform this step. Complete t
 
 .Prerequisites
 * You must have generated an init bundle containing secrets.
+* You must have created the `stackrox` project, or namespace, on the cluster where Central is installed. Using `stackrox` for the project is not required, but ensures that false positives for {product-title-short} processes are not reported when scanning your clusters.
```

Comment:
Is this prerequisite only for self-managed? I am not sure whether this is within a conditional block, as above.

If I understand correctly, cloud service customers do not have access to the cluster where Central is installed.

Reply:
This should actually read *...on the cluster where the secured cluster services will be installed.* So it applies to both.

Reply:
Aha, that makes sense!